### PR TITLE
Bridge automation

### PR DESCRIPTION
**Pull request by domenkozar**
Use Nix to build the bridge more reliably.

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -43,6 +43,7 @@ test_script:
   - copy log-config-prod.yaml daedalus\log-config-prod.yaml
   - cd daedalus
   - Echo %APPVEYOR_BUILD_VERSION% > build-id
+  - ps: Install-Product node 7
```
Comment (domenkozar) [inline appveyor.yml]: This needs to specify the platform

**Comment by rockbmb** [comment]
@domenkozar Would this PR still be helpful to you?

**Comment by domenkozar** [comment]
Not to me, but the frontend team. I'll get back to it some day in not so distant future :)

**Comment by neongreen** [comment]
Just in case: this is still WIP, right? If so, can you add [WIP] to the title? Because currently I've no idea what needs to be done with it.

**Comment by Pastafarianist** [comment]
Is this PR still relevant at all?

**Comment by domenkozar** [comment]
We still want to do something similar, but we can revisit this part in the future.

**Pull request closed without merging**